src/stories/OSCALControlImplementation.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import OSCALControlImplementation from "../components/OSCALControlImplementation";
+import { exampleComponents } from "../test-data/ComponentsData";
+import {
+  controlImplTestData,
+  exampleControl,
+  exampleImplNoStatements,
+} from "../test-data/ControlsData";
+
+export default {
+  title: "Components/Control Implementation",
+  component: OSCALControlImplementation,
+};
+
+const Template = (args) => <OSCALControlImplementation {...args} />;
+
+export const Default = Template.bind({});
+
+export const ImplementedStatements = Template.bind({});
+
+const exampleControls = [exampleControl];
+
+ImplementedStatements.args = {
```

Comment:
For consistency with the other components, lets rename this to `WithImplementedStatements`.